hledger/Hledger/Cli/BalanceView.hs

```diff
@@ -129,7 +129,7 @@ balanceviewReport BalanceView{..} CliOpts{command_=cmd, reportopts_=ropts, rawop
         unless (no_total_ ropts' || cmd=="cashflow") . mapM_ putStrLn $ -- TODO temp
           [ "Total:"
           , "--------------------"
-          , padleft 20 $ showMixedAmountWithoutPrice (getSum amt)
+          , padLeftWide 20 $ showamt (getSum amt)
```

Comment:
I switched to `leftPadWide` because for some reason, it would not pad the amounts if they were negative/red.  From the examples I tried, this did not affect the behavior in the normal non-colored cases.